src/_pytest/hookspec.py

```diff
@@ -198,7 +198,8 @@ def pytest_ignore_collect(path, config):
     This hook is consulted for all files and directories prior to calling
     more specific hooks.
 
-    Stops at first non-None result, see :ref:`firstresult`
+    Stops at first non-None result, see :ref:`firstresult`, i.e. you should
```

Comment:
```suggestion
    As any other :ref:`firstresult` hook, this stops at first non-`None` result. This hook
    can return `False` explicitly to prevent a file to be ignored by other hooks.
```

Reply:
I've pulled out the doc change.
There are more places using the same sentence, and this should be fixed across all of them, maybe adding and linking to an intermediate doc section then.